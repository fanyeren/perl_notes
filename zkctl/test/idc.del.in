#!/bin/bash
source $(dirname $0)/common

node_clear idc

./zkctl -w idc add --idc-id=jx --data='{"auto_start_new_service":"off","safe_mode":"off"}'
./zkctl -w idc add --idc-id=tc --data='{"auto_start_new_service":"off","safe_mode":"off"}'
echo "current 2 idc"
./zkctl -w idc list | sort
echo "after delete jx"
./zkctl -w idc del --idc-id=jx
./zkctl -w idc list | sort
