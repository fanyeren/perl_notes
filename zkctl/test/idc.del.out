current 2 idc
jx
tc
after delete jx
tc
